components/Formsy/PhoneInput.jsx

```diff
@@ -97,10 +113,12 @@ class PhoneInput extends Component {
     }
 
     this.props.onChange(this.props.name, value)
+    const isValid = this.checkPhone(value)
```

Comment:
If we implement a `validate` method, we don't have to add `isValid` anymore, as validation should work automatically.